cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -441,3 +457,27 @@ def test_ansible_group_id_in_payload(service_catalog, service_request, service):
     result_dict = json.loads(json_str[5].replace('", "', "").replace('\\"', '"').replace(
         '\\, "', '",').split('" ] } PLAY')[0])
     assert "group" in result_dict["manageiq"]
+
+
+@pytest.mark.provider([EC2Provider], scope="function")
+def test_embed_tower_exec_play_against_amazon(request, provider, setup_provider,
+        ansible_catalog_item, service, ansible_amazon_credential, service_catalog):
+    with update(ansible_catalog_item):
+        ansible_catalog_item.provisioning = {
+            "playbook": "list_ec2_instances.yml",
+            "cloud_type": "Amazon",
+            "cloud_credential": ansible_amazon_credential.name
+        }
+
+    def _revert():
+        with update(ansible_catalog_item):
+            ansible_catalog_item.provisioning = {
+                "playbook": "dump_all_variables.yml",
+                "cloud_type": "<Choose>"
+            }
+
+    request.addfinalizer(_revert)
+    service_request = service_catalog.order()
+    service_request.wait_for_request(method="ui", num_sec=200 * 60, delay=20)
```

Comment:
I'm thinking this is a typo for a 20 minute timeout? 200 minutes is far too long.